### DISCUSSION TITLE
Cannot access web UI when connected as STA

### DISCUSSION THREAD

**Discussion by laffineuse** [thehighnibble/thehighnibble.github.io#175] [IMSAI8080esp replica]
When my IMSAI is configured for AP access, the imsai8080 SSID is displayed in my list of available networks and I can access the web UI via the IP address.  However whenever it is configurated via STA, I can no longer access the UI via the IP address or the http://imsai8080.  What am I missing?

**Comment by udo-munk** [comment]
The address then is http://imsai8080.local or http://imsai8080.net. Depends on what your WiFi router does exactly. It is possible that it won't support a .local network, in this case lookup the IP address in the router and use that in the URL instead of a name.
